src/ProjectEnvironmentConfiger.ts

```diff
@@ -23,22 +23,31 @@ const ioTContainerizedProjectModule =
     typeof import('./Models/IoTContainerizedProject');
 
 export class ProjectEnvironmentConfiger {
+  // Configuration command will scaffold configutation
+  // files for project and WILL NOT open an Embedded Linux project in remote by default.
   async configureProjectEnvironment(
       context: vscode.ExtensionContext, channel: vscode.OutputChannel,
       telemetryContext: TelemetryContext,
-      platform: PlatformType = PlatformType.Unknown) {
+      platform: PlatformType = PlatformType.Unknown, openInNewWindow = false) {
     // Only create project when not in remote environment
     const isLocal = RemoteExtension.checkLocalBeforeRunCommand(context);
     if (!isLocal) {
       return;
     }
 
-    let openInNewWindow = false;
-    // If current window contains other project, open the created project in new
-    // window.
-    if (vscode.workspace.workspaceFolders &&
-        vscode.workspace.workspaceFolders.length > 0) {
-      openInNewWindow = true;
+    if (!vscode.workspace.workspaceFolders) {
+      const message =
+          'No folder is currently open in Visual Studio Code. Please select a folder first.';
```

Comment:
You have not yet opened a folder in Visual Studio Code. Please select a folder first.